modules/plugin/coverage-multidim/netcdf/src/test/java/org/geotools/coverage/io/netcdf/NetCDFUnitParserTest.java

```diff
@@ -160,15 +160,15 @@ public void reset() {
         public void testDU() throws Exception {
             // could not find a way to make this work with the above test, so doing it another way
             Unit<?> unit = NetCDFUnitFormat.parse("DU");
-            assertEquals("(1/m²)*446.2·μmol", unit.toString());
+            assertEquals("μmol·(1/m²)*446.2", unit.toString());
         }
 
         @Test(expected = MeasurementParseException.class)
         public void testIsolation() {
             // the normal instance should be isolated, the configuration of the the NetCDF
             // unit parse should not affect the normal parser
             SimpleUnitFormat instance = SimpleUnitFormat.getInstance();
-            instance.parse("degree");
+            instance.parse("Degree");
```

Comment:
Any chance you can build tolerance for "degree" in the library, to avoid backward compatibility issues?

Reply:
My mistake , "degree" still works. I revert my change.